submissions/2024-09-18-OpenFF-Sulfur-Hessian-Training-Coverage-Supplement-v1.0/generate.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+
+import numpy as np
+import qcportal  # noqa avoid zstd disaster
+from openff.qcsubmit.factories import BasicDatasetFactory
+from openff.qcsubmit.results import OptimizationResultCollection
+from openff.qcsubmit.results.filters import (
+    RecordStatusEnum,
+    RecordStatusFilter,
+)
+from openff.qcsubmit.utils import _CachedPortalClient, portal_client_manager
+from openff.toolkit import ForceField
+from qcaide import Submission
+from qcportal.singlepoint import SinglepointDriver
+
+# Load config file and force field
+ff = ForceField("openff-2.1.0.offxml")
+config = Submission.from_toml("opt.toml")
+
+client = _CachedPortalClient("https://api.qcarchive.molssi.org", ".")
+opt = OptimizationResultCollection.from_server(
+    client,
+    datasets=["OpenFF Sulfur Optimization Training Coverage Supplement v1.0"],
+)
+
+print(f"Retrieved {opt.n_results} results")
+
+with portal_client_manager(lambda _: client):
+    recs = opt.filter(
+        RecordStatusFilter(status=RecordStatusEnum.complete)
```

Comment:
Could you also please add a ConnectivityFilter for records where the connectivity may have changed?

Reply:
Will do!

Reply:
Thank you!!